src/main/java/org/opensearch/commons/model2/action/ExecuteMonitorAction.java

```diff
@@ -0,0 +1,13 @@
+package org.opensearch.commons.model2.action;
+
+import org.opensearch.action.ActionType;
+
+public class ExecuteMonitorAction extends ActionType<ExecuteMonitorResponse> {
+
+    public static final String ALERTING_NAME = "cluster:admin/opendistro/security_analytics/monitor/execute";
+    public static final ExecuteMonitorAction ALERTING_INSTANCE = new ExecuteMonitorAction(ALERTING_NAME);
```

Comment:
Can we only move the ACTION_NAME here in the common-utils while the Action class can still be retailed by Alerting. That way ownership remains clear, while the action name is available to clients.